src/WordPress/WpEventManager.php

```diff
@@ -238,6 +258,39 @@ public function __construct($enablePropagation = false)
         }
     }
 
+    /**
+     * Retrieves the argument count value to pass to the native WordPress action and filter hooking functions.
+     *
+     * @since [*next-version*]
+     *
+     * @return int
+     */
+    protected function _getNumArgs()
```

Comment:
Is something preventing the extraction of this into a trait?

Reply:
Weird question to ask :P Answer: no

But if you meant to ask me to extract it into a trait, I'd probably answer with: `10-4`

---

_Already on it_